core/zip/src/Compression.cxx

```diff
@@ -25,24 +24,27 @@ namespace ROOT {
     return algo * 100 + compressionLevel;
   }
 
-  int CompressionSettings(ROOT::ECompressionAlgorithm algorithm,
-                          int compressionLevel)
+// Avoid spitting warnings for the use of ECompressionAlgorithm, as this function is itself deprecated
+#pragma GCC diagnostic push
```

Comment:
So this is a problem only for GCC? We should also check MSVC.

Reply:
Yes, I'll check the windows tests when they are done.